std_msgs/Float64 rho_12
std_msgs/Float64 rho_13
std_msgs/Float64 rho_23
